pkgs/top-level/all-packages.nix

```diff
@@ -33434,7 +33434,10 @@ with pkgs;
 
   sqsh = callPackage ../development/tools/sqsh { };
 
-  sumneko-lua-language-server = callPackage ../development/tools/sumneko-lua-language-server { };
+  sumneko-lua-language-server = callPackage ../development/tools/sumneko-lua-language-server
+    (lib.optionalAttrs (stdenv.cc.isClang && lib.versionOlder stdenv.cc.version "9") {
+      stdenv = overrideCC stdenv llvmPackages_9.clang;
+    });
```

Comment:
Doesn't build: https://gist.github.com/veprbl/b2b001fcec7751a255736e9e11e9f180

Reply:
Thanks for testing. It is the same output as without my PR. I do not know how to fix it.